tests/baselines/vit_base_patch16_224_in21k.json

```diff
@@ -0,0 +1,36 @@
+{
+    "beans": {
+        "num_train_epochs": 5,
+        "eval_batch_size": 8,
+        "distribution": {
+            "single_card": {
+                "learning_rate": 2e-5,
+                "train_batch_size": 8,
+                "eval_accuracy": 0.985,
+                "train_runtime": 92.8927,
+                "extra_arguments": [
```

Comment:
do we have image per sec throughput measured?

Reply:
I can measure it. So far we have not used it as a regression metric in CI, but we can add it in another PR.